incubator/kubeless/templates/clusterRole.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kubeless-controller-deployer
```

Comment:
Resource names should use the `fullname` template. Apply everywhere.

Reply:
Done except for the kafka service. Right now the kafka service name is hardcoded in kubeless.